HeartbeatLogging/Sources/StorageFactory.swift

```diff
@@ -14,7 +14,7 @@
 
 import Foundation
 
-private enum Constants {
+enum Constants {
```

Comment:
`private` access removed since these constants are now used in HeartbeatLogging/Sources/HeartbeatLoggingTestUtils.swift

Reply:
Make private again since they have been copied over to new HeartbeatLoggingTestUtils target